index.html

```diff
@@ -1,3 +1,29 @@
-<html>
-  
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <ul class="nav-bar">
+            <li><a href="#">Porfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">info</a></li>
+        </ul>
+    </header>
+    <main class="main-content">
+        <h1 class="heading">Hey there I,m Bunbee.<br>I'm a software developer</h1>
+        <p>I can help you build a product, feature or website. Look thrrough some of my work and experience! If you like what you see and have a project you need coded, dont hesitate to contact me.</p>
+        <ul class="footer">
+            <li><a href="https://github.com/mutinhiri">github</a></li>
+            <li><a href="www.linkedin/mutinhiri">Linkedin</a></li>
+            <li><a href="twitter.com/bunbee">Twitter</a></li>
+        </ul>
+    </main>
```

Comment:
- [ ] This should be **`section`** tag instead of main tag.
![gdsghsdc](https://user-images.githubusercontent.com/61562626/129913367-1b35b09c-7375-4fb5-a571-f6d74bb374a3.jpeg)